tests/probes/uname/test_probes_uname.xml.sh

```diff
@@ -846,7 +846,6 @@ cat <<EOF
       <os_name>`uname -s`</os_name>
       <os_release>`uname -r`</os_release>
       <os_version>`uname -v`</os_version>
-      <processor_type>`uname -p`</processor_type>
```

Comment:
This is actually a valid test for the uname_probe ``<processor_type>`uname -p`</processor_type>``. 
Don't think that it should be removed. See https://github.com/OVAL-Community/OVAL/blob/master/oval-schemas/unix-definitions-schema.xsd#L2526

Reply:
The problem is it's not a POSIX compliant uname option an behaves differently on Fedora than everywhere else.  https://unix.stackexchange.com/questions/307955/uname-p-i-are-unknown

Reply:
Correct. The test should be updated for Debian returning `unknown` for `uname -p`.

Reply:
Then the root cause may be a bug that `src/OVAL/probes/unix/uname_probe.c` processor_type and machine_class as interchangeable.  On Debian `uname -m` and `uname -p` do not give the same output.  I believe this uname test is failing not just because `uname -p` is returning `unknown`, but because `unknown` doesn't match the return from `uname -m`.  My result.xml contains processor_type references to `unknown` from inline `uname` commands and `x86_64` from `system_data`.

Reply:
Problem is

Debian 10
```
ekolesni@debian10-uni:~/openscap$ uname -i
unknown
ekolesni@debian10-uni:~/openscap$ uname -p
unknown
ekolesni@debian10-uni:~/openscap$ uname -m
x86_64
ekolesni@debian10-uni:~/openscap$ 
```
Fedora 30:
```
[ekolesni@ektp ~]$ uname -p
x86_64
[ekolesni@ektp ~]$ uname -m
x86_64
[ekolesni@ektp ~]$ uname -i
x86_64
[ekolesni@ektp ~]$ 
```
But, in the probe `machine` used twice, as there is no `processor` entity in the `uname` call:
```
        uname(&buf);

        item = probe_item_create(OVAL_UNIX_UNAME, NULL,
                                 "machine_class",  OVAL_DATATYPE_STRING, buf.machine,
                                 "node_name",      OVAL_DATATYPE_STRING, buf.nodename,
                                 "os_name",        OVAL_DATATYPE_STRING, buf.sysname,
                                 "os_release",     OVAL_DATATYPE_STRING, buf.release,
                                 "os_version",     OVAL_DATATYPE_STRING, buf.version,
                                 "processor_type", OVAL_DATATYPE_STRING, buf.machine,
                                 NULL);
```
So,
```suggestion
      <processor_type>`uname -m`</processor_type>
```
should do the trick.